packages/@ember/-internals/glimmer/lib/helpers/mut.ts

```diff
@@ -34,6 +37,10 @@ import { INVOKE, UPDATE } from '../utils/references';
   Additionally, the `mut` helper can be combined with the `action` helper to
   mutate a value. For example:
 
+  ```handlebars
+  <MyChild @childClickCount={{this.totalClicks}} @click-count-change={{action (mut totalClicks))}} />
```

Comment:
This example needs to be revised (both syntax).

Reply:
not sure what you mean here.